pymc4/random_variables/continuous.py

```diff
@@ -5,45 +5,1293 @@
 Implements random variables not supported by tfp as distributions.
 """
 
-# FIXME all RandomVariable classes need docstrings
 # pylint: disable=undefined-all-variable
-import sys
 import tensorflow_probability as tfp
+import tensorflow as tf
 from tensorflow_probability import distributions as tfd
 
 from .random_variable import RandomVariable
+from typing import NewType, Union, Sequence
+import numpy as np
+
+
+TensorLike = NewType("TensorLike", Union[Sequence[int], Sequence[float], int, float])
```

Comment:
Why tf.Tensor is not included? Is it redundant?

Reply:
Sequence[float] and Sequence[int] covers all Tensor containers, IIRC.